tests/core/daemon/test_daemon.py

```diff
@@ -169,6 +170,100 @@ async def reader(ws, queue):
     assert blockchain_state_found
 
 
+@pytest.mark.asyncio
+async def test_is_running_wildcard():
```

Comment:
I think this could be several separate tests without inducing much overhead.  Move the classes to the global scope, maybe make a fixture for each daemon form used here.  This allows granular results and also complete results even when the first assertion fails.

Reply:
Added separate tests.